thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/AnomalyDetectionUtils.java

```diff
@@ -12,4 +16,23 @@
   public static long getBucketInMillis(int bucketSize, TimeUnit bucketUnit) {
     return bucketUnit.toMillis(bucketSize);
   }
+
+  private static Splitter SEMICOLON_SPLITTER = Splitter.on(";").omitEmptyStrings();
+  private static Splitter EQUALS_SPLITTER = Splitter.on("=").omitEmptyStrings();
+
+  /**
+   * Decode properties string to into Hashmap
+   * @param propStr: property string which is encoded using com.linkedin.anomalydetection.utils.StringUtil.encodeCompactedProperties
```

Comment:
You'd rather move the whole files here. 
StringUtil.java, you do not need to leave a comments like this in a open source project.